### DISCUSSION TITLE
SONY STRDN-1080 2 HDMI OUTPUTS

### DISCUSSION THREAD

**Discussion by thepuncake** [awawa-dev/HyperHDR#977] [Q&A]
What I'm trying to accomplish here is have hyperhdr working on all hdmi sources supporting hdr where hdr is supported. So HDR works on my PC for example until I turn on the second hmdi output on the AVR which is plugged in the ms2130. Then the first output converts to SDR. I've tried connecting a Feintech VSP01202 splitter on the second output which then goes to the grabber with not much difference in results. If I connect a 4k chromecast on the input of the splitter and 1 output to avr input and another to the grabber everything works as intended but that limits me to a single source. Is there any way to get this to work with all my sources without spending too much? Appreciate the feedback.

**Comment by medapayne** [comment]
Hello i asking the same question yesterday : 

https://github.com/awawa-dev/HyperHDR/discussions/976

 i hope we have a answer ;-)

**Reply by thepuncake** [reply]
Yeah would be nice, been fiddling with this for a while now. @awawa-dev could this be resolved if I flash the experimental firmware on to the ms2130? Thanks!

**Reply by awawa-dev** [reply]
For me, HDMI AVR splitters are useless in typical ambient lighting setups due to their very limited capabilities. Also, the Feintech splitter looks very pale against the competition. However, we would need opinions from users who use similar setups, because only in practice can we verify them.

**Reply by thepuncake** [reply]
Appreciate your answer, I'm asking about the firmware because reading this (https://github.com/awawa-dev/HyperHDR/discussions/729#discussioncomment-8126044) I see that MS2130 reports HDR support after the experimental firmware is flashed and the EDID is changed. Isn't this very likely to be the problem? Is it worth a shot? Thanks again and I can't give you enough props for your work! :)

**Reply by awawa-dev** [reply]
Experimental firmware for MS2130 has almost exactly MS2109 EDID which fixes some compatibility issues and add missing modes like 1024p23.9 + defines HDR and LLDV support. Only EDID has been changed, the core MS2130 firmware is the same. If you follow the procedure then you can backup your current firmware and restore it back later.

**Reply by thepuncake** [reply]
Cool, cheers, is there a chance this could solve my problem though? Because I can get HDR from both outputs simultaneously if I connect another monitor there for example. I guess what I'm asking is there a chance that HDR doesn't work because ms2130 is reporting the wrong EDID to the AVR? And can that potentially be resolved with the experimental software? I'm not looking for definitive instructions here nor will I hold you responsible if something doesn't work :) Need a go from you if you think there is a 10% chance it would work!

**Reply by awawa-dev** [reply]
Currently if you have ms2130 original firmware it wont report HDR capabilities (its EDID is not wrong - formally it's just a HDMI 1.4 grabber so no HDR) and HDMI splitter must force it to accept it by ignoring its EDID. Other devices will probably obey no-HDR EDID and wont force it.
For me experimental firmware fixed an issue with Ezcoo splitter: https://github.com/awawa-dev/HyperHDR/discussions/729#discussioncomment-8126042 the opinions of other users are divided, so no guarantee.
